Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
       SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
       export SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN && \
       pnpm run -r build-with-sourcemaps --release $SENTRY_RELEASE
+# Comment above and uncomment below to build the image locally (not in CI)
+# RUN pnpm --filter=app-builder run build
+ENV NODE_ENV=production
```

Comment:
Honnestly, I think we could have two dockerfile : one for the CI and one for the local build.

Reply:
hmm, do we ?
I'm assuming us as developers will usually develop in VScode not in a local docker image.
On the other hand our open source users will use the same docker image as us in prod.

Reply:
I mean, we can add an alternative dockerfile for local docker image building when debugging (as I had to do this week) - but I don't expect it to be used much